### ISSUE TITLE
Set temprature

### ISSUE THREAD

**Issue by lakpriya1s** [daviddaytw/react-native-transformers#8]
**Is your feature request related to a problem? Please describe.**
I'm working with react-native-transformers to integrate transformer models, and I need to control the temperature parameter when using models like Qwen1.5-0.5B-Chat. Currently, there's no built-in support or clear documentation on how to set the temperature for inference in this library, which limits my ability to fine-tune the creativity and randomness of generated responses.

**Describe the solution you'd like**
I'd like native support in react-native-transformers for setting the temperature parameter during model inference. This could be added as part of the model's options configuration, similar to how it's defined below:

```
{
  "name": "Qwen1.5-0.5B-Chat",
  "model": "lakpriya/Qwen1.5-0.5B-Chat-fraud-detection-onnx_quantized",
  "onnx_path": "model_quantized.onnx",
  "options": {
    "externalData": true,
    "fileExtension": ".data",
    "temperature": 0.1,
  }
}
```

**Describe alternatives you've considered**
- Manually adjusting the logits after model inference, but this adds unnecessary complexity and defeats the purpose of using a high-level framework.
- Forking the library to patch in support, but this is not ideal for long-term maintenance or for community usage.

**Additional context**
- penalty_alpha: 0.5,
- top_k: 4,
- repetition_penalty: 1.01

**Comment by daviddaytw** [comment]
This is interesting.
